pkg/runtime/cli.go

```diff
@@ -228,9 +228,18 @@ func FromFlags() (*DaprRuntime, error) {
 		concurrency = *appMaxConcurrency
 	}
 
-	appPrtcl := string(HTTPProtocol)
-	if *appProtocol != string(HTTPProtocol) {
-		appPrtcl = *appProtocol
+	var appProtocol string
```

Comment:
Note this changes actually enforces protocol to be one of "http" or "grpc" (case-insensitive)